reference/pipeline/stages/index.md

```diff
@@ -13,10 +13,27 @@ sidebar:
 This article describes the currently-supported stages that you can add to your
 Spinnaker pipelines.
 
+Note that when you're creating a pipeline, you probably won't see every stage
+that's listed here: you'll only see the stages that Spinnaker supports on your
+provider.
```

Comment:
Are there other factors that determine which stages populate the list, besides provider? 
@lwander

Reply:
None that are relevant here - I think some are hidden by "netflixMode", a feature flag, but that's arguably not important.